WBooks/Screens/BookDetails/CommentCell/CommentCellViewModel.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  CommentCellViewModel.swift
+//  WBooks
+//
+//  Created by Nacho 2 on 01/05/2020.
+//  Copyright © 2020 Wolox. All rights reserved.
+//
+
+import UIKit
+
+class CommentCellViewModel {
+    let commentModel: Comment
+    
+    init(commentModel: Comment) {
+        self.commentModel = commentModel
+    }
+    
+    var image: String? {
+        guard let imageName = commentModel.image, imageName.isNotEmpty else { return nil }
+        return  imageName
```

Comment:
It's better this, If you want, change BookDetailsViewModel image with this, so you don't use the `!`